docs/source/developing/writing-docs.adoc

```diff
@@ -25,23 +20,26 @@ maintained https://en.wikipedia.org/wiki/AsciiDoc[Asciidoc] files.
 Per default the documentation is build in a Docker container. This way
 you avoid having to install all required dependencies on your
 development machine. All you need is a running Docker daemon. In case
-you don't have one, use Minishift itself. See (TODO reference to the
-Minishift as Docker daemon from README. Needs to be extracted).
+you don't have one, use Minishift itself. See also
+link:../command-ref/minishift_docker-env{outfilesuffix}[`minishift docker-env`].
 
```

Comment:
Ok for now. Once the other outstanding pull request is merged, I'll repoint the link anyways.